monitoring/uss_qualifier/resources/interuss/mock_uss.py

```diff
@@ -29,12 +29,20 @@ def get_status(self) -> fetch.Query:
         return fetch.describe_query(resp, initiated_at)
 
     # TODO: Add other methods to interact with the mock USS in other ways (like starting/stopping message signing data collection)
+    def start_msg_sign_recording(self) -> fetch.Query:
```

Comment:
Adding these methods resolves the TODO above, so we can delete that line in this PR.

Reply:
Sure. Will do.